test/boost/cql_query_test.cc

```diff
@@ -1121,24 +1121,20 @@ SEASTAR_TEST_CASE(test_range_deletion_scenarios_with_compact_storage) {
 
 SEASTAR_TEST_CASE(test_invalid_range_deletion) {
```

Comment:
I think this test deserves a comment and/or a name change. Maybe it used to test "invalid range deletion" when UNBOUNDED_RANGE_TOMBSTONE is disabled, but if I understand correctly it no longer has any invalid cases?

Reply:
I renamed it to test_range_deletion.

Reply:
Actually we already have test_range_deletion_scenarios so I will just delete this test.